wazimap_ng/profile/serializers/utils.py

```diff
@@ -43,7 +43,8 @@ def get_indicator_data(profile, indicators, geographies, version):
                 content_type=F("indicator__profileindicator__content_type"),
                 chart_type=F("indicator__profileindicator__chart_type"),
                 choropleth_range=F("indicator__profileindicator__choropleth_range"),
-                enable_linear_scrubber=F("indicator__profileindicator__enable_linear_scrubber")
+                enable_linear_scrubber=F("indicator__profileindicator__enable_linear_scrubber"),
+                pi_order=F("indicator__profileindicator__order")
```

Comment:
To be consistent I think this should be
```suggestion
                profile_indicator_order=F("indicator__profileindicator__order")
```